digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/dataimport/CsvDataImporterSpec.scala

```diff
@@ -355,9 +355,131 @@ class CsvDataImporterSpec extends AuthenticatedApiSpec with BeforeAndAfter {
 
     trafficSignCsvImporter.importTrafficSigns(invalidCsv, Set()) should equal(trafficSignCsvImporter.ImportResult(
       notImportedData = List(trafficSignCsvImporter.NotImportedData(
-        reason = "Tried to create in an unauthorized municipality",
+        reason = "Unauthorized Municipality Or RoadLind inexistent near of Asset",
```

Comment:
Probably could be rewritten to "Unathorized Municipality or nonexistent roadlink near asset position" or something similar